test/common/testing_idp.go

```diff
@@ -328,25 +349,29 @@ func createKeycloakUsers(ctx context.Context, client dynclient.Client, keycloakC
 		postfix++
 	}
 
-	err := createOrUpdateKeycloakUserCR(ctx, client, testUsers)
+	err := createOrUpdateKeycloakUserCR(ctx, client, testUsers, installationName)
 	if err != nil {
 		return fmt.Errorf("error occurred while creating or updating keycloak user: %w", err)
 	}
 
 	return nil
 }
 
-func createOrUpdateKeycloakUserCR(ctx context.Context, client dynclient.Client, testUsers []TestUser) error {
+func createOrUpdateKeycloakUserCR(ctx context.Context, client dynclient.Client, testUsers []TestUser, installationName string) error {
 	// create rhmi developer users from test users
 	for _, user := range testUsers {
 		keycloakUser := &v1alpha1.KeycloakUser{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      fmt.Sprintf("%s-%s", TestingIDPRealm, user.UserName),
-				Namespace: fmt.Sprintf("%srhsso", NamespacePrefix),
+				Namespace: RHSSOProductNamespace,
 			},
 		}
 
 		_, err := controllerutil.CreateOrUpdate(ctx, client, keycloakUser, func() error {
+			keycloakUser.Annotations = map[string]string{
+				"integreatly-namespace": RHMIOperatorNamespace,
+				"integreatly-name":      installationName,
+			}
```

Comment:
Add annotations to created CRs to have reconciler do a reconcile due to change in watched CRDs

Operator now only reconciles every 5 minutes if the installation is complete. Because of this, tests that depend on a reconcile run (e.g. Custom broker) may fail simply due to timeout or failure as the operator was not reconciling

* https://github.com/integr8ly/integreatly-operator/blob/master/pkg/controller/installation/installation_controller.go#L361-L365

The annotation addition and the recreation of the testing idp keycloak realm should help but only runs for 1 reconcile, but tests might still fail due to this